SetupBLT.cmake

```diff
@@ -82,6 +82,13 @@ if (NOT BLT_LOADED)
   endif()
 
   ################################
+  # Enable CXX required by GTEST
+  ################################
+  if(ENABLE_GTEST)
```

Comment:
Not sure about this one.

I think we'd probably want the logic to go the other way.
I.e. rather than enabling ``CXX`` when ``GTEST`` is on, it should be a config error if ``GTEST`` is on when ``CXX`` is not available, and there should be an appropriate error message.
(Although, perhaps this could be valid for C projects that want to use gtest?)

Can others please weigh in here?

Reply:
I will note that testing C with GTest is perfectly valid/reasonable. I was tempted to implement it the way you were suggesting, but I didn't want to cause unexpected behavior (i.e. this will break backwards compatibility if people rely on GTest for C codes and haven't been explicitly enabling it)

Reply:
Was this brought on by an error you encountered?

Reply:
I think it was this: #105

Reply:
@white238 yes #105. If you create a C or Fortran only minimal project CMake (at least the latest the latest stable version) throws an error. It’s possible it could be a CMake bug. I didn’t localize it beyond enabling gtest

Reply:
Sorry coming in to this at the end due to a vacation (Disneyland was fun as always). This looks great.

Reply:
Like @kennyweiss  -- I would prefer we add an error message if  ENABLE_GTEST is on, but CXX isn't enabled. 

(one that gives folks a chance to know whats wrong vs any cmake error they will encounter otherwise) 

Even though you can use gtest for testing c, it's a c++ framework and folks are going to have to use a c++ compiler to compile tests.